app/src/pages/CardSubmission/WaitingForSubmissions.tsx

```diff
@@ -37,77 +37,78 @@ function WaitingForSubmissions() {
+                  {submittedSoFar}/{`${total} cards so far`}
+                </Typography>
+              </Grid>
+            </>
+          ),
+          [WaitingForSubmissionsState.SubmittedAssign]: (
             <Grid item>
-              {currentPlayer.role === PlayerRole.Host ? (
-                <Button
-                  size="large"
-                  variant="contained"
-                  color="primary"
-                  onClick={async () => {
-                    const players = teamsWithSequence(currentGame.players)
-                    await updateAllPlayers({
-                      variables: {
-                        gameId: currentGame.id,
-                        players: players.map(({ id, team, team_sequence }) => ({
-                          id,
-                          team,
-                          team_sequence,
-                        })),
-                      },
-                    })
-                    updateGameState({
-                      variables: {
-                        id: currentGame.id,
-                        state: GameStateEnum.TeamAssignment,
-                      },
-                    })
-                  }}
-                >
-                  Assign Teams
-                </Button>
-              ) : null}
+              All players submitted! As the host, you can now assign teams.
             </Grid>
-          )}
-        </>
-      ) : (
-        <>
-          <Grid item container justify="center">
-            Just waiting for everyone else...<div style={{ width: 4 }}></div>
-            {waitingForPlayers.map((player) => (
-              <>
-                <PlayerChip username={player.username || ""}></PlayerChip>
-                <div style={{ width: 4 }}></div>
-              </>
-            ))}
-          </Grid>
-          <Grid item>
-            <Typography variant="h5">
-              {submittedSoFar}/{`${total} cards so far`}
-            </Typography>
-          </Grid>
-          <Grid item>
-            <div style={{ marginTop: 50 }}>
-              <Fishbowl></Fishbowl>
-            </div>
+          ),
+          [WaitingForSubmissionsState.SubmittedWait]: (
+            <Grid item>
+              All players submitted, ${submittedSoFar} cards in total. Now we
+              are waiting on the host to start the game!
+            </Grid>
+          ),
+        }[state]
+      }
+      <Grid item container direction="column" spacing={2} alignItems="center">
+        {unscreenedCards.map((card, index) => (
+          <Grid item key={index}>
+            <BowlCard>{card.word}</BowlCard>
           </Grid>
-        </>
-      )}
+        ))}
+      </Grid>
+      {WaitingForSubmissionsState.SubmittedAssign === state ? (
+        <AssignTeamsButton />
+      ) : WaitingForSubmissionsState.Waiting === state &&
+        !unscreenedCards.length ? (
+        <div style={{ marginTop: 50 }}>
+          <Fishbowl />
+        </div>
+      ) : null}
```

Comment:
In its current state, this would enable hosts to assign teams, even if they didn't explicitly screen every card. Maybe this is preferable, i.e. perhaps we only expose the ability to "reject" cards for simplicity?

We also might consider just _always_ displaying the fishbowl graphic for players; otherwise, I think the states could jump back and forth depending on when the host rejects cards?

Reply:
i think always fishbowl graphic, sounds simpler/cleaner